src/utils/payload.js

```diff
@@ -0,0 +1,18 @@
+import fs from 'fs';
+
+export function loadPayloadFile(path) {
```

Comment:
doesn't look like you import this anywhere. Could you just make this a normal function?